var/spack/repos/builtin/packages/hdf5-vol-async/package.py

```diff
@@ -27,7 +27,7 @@ class Hdf5VolAsync(CMakePackage):
     depends_on("hdf5@1.13: +mpi +threadsafe")
 
     def setup_run_environment(self, env):
-        env.set("HDF5_PLUGIN_PATH", self.spec.prefix)
+        env.set("HDF5_PLUGIN_PATH", join_path(self.spec.prefix, "lib"))
```

Comment:
Looks good.

Did you know you could simply do the following instead?

```suggestion
        env.set("HDF5_PLUGIN_PATH", self.spec.prefix.lib)
```

Reply:
@jeanbez would you please update your PR following @tldahlgren 's suggestion?

Reply:
@hyoklee sure!